### DISCUSSION TITLE
The stars page looks too much like a webpage

### DISCUSSION THREAD

**Discussion by leap0x7b** [community/community#21052] [Mobile]
![Screenshot_20220722-112304.png](https://user-images.githubusercontent.com/76893443/180362366-e9ec1902-2c4f-4fb2-8192-4d4e8dd2c8eb.png)

I know it might look like one but it actually uses native Android controls

**Comment by iskae** [comment]
Hi @leapofazzam123 👋 

Thanks for reaching out 😊 

> I know it might look like one but it actually uses native Android controls

Yep, you are right 👍 This screen is actually implemented using native Android components 😊 

Can you elaborate a bit more on which part looks like a webpage? 😅 We would love to offer a platform specific experience to our users as much as possible, therefore if you have any suggestions we would love to follow up on them 😊 

Cheers!

**Reply by leap0x7b** [reply]
the top

**Reply by iskae** [reply]
Hi @leapofazzam123 👋 

Thanks for your response! 

I will be reaching out to our designers and discuss internally if we can make improvements for this 👍 

Cheers!
